internal/provider/namespaces_datasource.go

```diff
@@ -0,0 +1,445 @@
+							ElementType: types.StringType,
+						},
+						"accepted_client_ca": schema.StringAttribute{
+							Computed:    true,
+							Description: "The Base64-encoded CA cert in PEM format that clients use when authenticating with Temporal Cloud.",
+						},
+						"retention_days": schema.Int64Attribute{
+							Computed:    true,
+							Description: "The number of days to retain workflow history. Any changes to the retention period will be applied to all new running workflows.",
+						},
+						"certificate_filters": schema.ListNestedAttribute{
+							Computed:    true,
+							Optional:    true,
+							Description: "A list of filters to apply to client certificates when initiating a connection Temporal Cloud. If present, connections will only be allowed from client certificates whose distinguished name properties match at least one of the filters.",
+							NestedObject: schema.NestedAttributeObject{
+								Attributes: map[string]schema.Attribute{
+									"common_name": schema.StringAttribute{
+										Computed:    true,
+										Description: "The certificate's common name.",
+									},
+									"organization": schema.StringAttribute{
+										Computed:    true,
+										Description: "The certificate's organization.",
+									},
+									"organizational_unit": schema.StringAttribute{
+										Computed:    true,
+										Description: "The certificate's organizational unit.",
+									},
+									"subject_alternative_name": schema.StringAttribute{
+										Computed:    true,
+										Description: "The certificate's subject alternative name (or SAN).",
+									},
+								},
+							},
+						},
+						"codec_server": schema.SingleNestedAttribute{
+							Optional:    true,
+							Computed:    true,
+							Description: "A codec server is used by the Temporal Cloud UI to decode payloads for all users interacting with this namespace, even if the workflow history itself is encrypted.",
+							Attributes: map[string]schema.Attribute{
+								"endpoint": schema.StringAttribute{
+									Computed:    true,
+									Description: "The endpoint of the codec server.",
+								},
+								"pass_access_token": schema.BoolAttribute{
+									Computed:    true,
+									Description: "If true, Temporal Cloud will pass the access token to the codec server upon each request.",
+								},
+								"include_cross_origin_credentials": schema.BoolAttribute{
+									Computed:    true,
+									Description: "If true, Temporal Cloud will include cross-origin credentials in requests to the codec server.",
+								},
+							},
+						},
+						"endpoints": schema.SingleNestedAttribute{
+							Computed:    true,
+							Description: "The endpoints for the namespace.",
+							Attributes: map[string]schema.Attribute{
+								"web_address": schema.StringAttribute{
+									Description: "The web ui address.",
```

Comment:
```suggestion
									Description: "The web UI address.",
```

Reply:
As with `aws_private_link_info` I admittedly grabbed the description from the proto model comment 😅

Reply:
whoops - we'll go fix up the proto docs too haha